doc/admin-guide/files/records.config.en.rst

```diff
@@ -3845,6 +3845,13 @@ Sockets
    exception being if you run Traffic Server with a protocol plugin, and would
    like for it to not support HTTP requests at all.
 
+.. ts:cv:: CONFIG proxy.config.http.allow_half_open INT 1
+   :reloadable:
+   :overridable:
+
+   Turn on or off support for connection half open for client side. Default is on, so
+   after client sends FIN, the connection is still there.
+
```

Comment:
Please inform the end user that it is dangerous to turn it off. It can only be turned off in special scenario.